### ISSUE TITLE
Redirection no longer works for private and Jetpack blogs

### ISSUE THREAD

**Issue by tpenguinltg** [tpenguinltg/wpcom-edit-post-redirect.user.js#1]
The fallback link that the script scrapes is no longer useful. Figure out how to request with authentication like in the conditional comment:

``` javascript
    if(!blogid) {
      window.location = currentUser.primary_blog.url + "/wp-admin/post-new.php";
    }
    jQuery.ajax( {
      type: 'GET',
      url: "https://public-api.wordpress.com/rest/v1.1/sites/" + blog_id,
      dataType: 'jsonp',
      success:  function(data) {
        if ( 'new' == post_id ) {
          window.location = data.URL + "/wp-admin/post-new.php";
        } else {
          window.location = data.URL + "/wp-admin/post.php?post=" + post_id + "&action=edit";
        }
      }
    }
    );    
```

**Comment by tpenguinltg** [comment]
Consider scraping the sidebar, checking against the `data-blogid` attribute and grabbing the `data-blogurl` attribute.

**Comment by tpenguinltg** [comment]
Even better, no need for checking. Grab the `data-blogurl` of the element that matches the query selector `div.site.blog-select-click.is-selected` using the `jQuery` object and build the URL.

**Comment by tpenguinltg** [comment]
Fixed in v1.3.0.
